### PR TITLE
add support for the keepassx program on linux

### PR DESCRIPTION
**Pull request by fidgetingbits**

### CODE REVIEW

File: apps/linux/keepassx.talon [hunk 1]
```diff
@@ -0,0 +1,27 @@
+os: linux
+#app: /KeePassX/
+#app: /eepassx/
+# XXX for some reason the app: field doesn't match atm
```
Comment (knausj85) [inline apps/linux/keepassx.talon:4]: Just curious. Is the issue with app a bug with talon, or is keepassx one of those apps with a bad name (many things on Windows are 'ApplicationHost.dll' or something).

Comment (fidgetingbits) [reply apps/linux/keepassx.talon:4]: I believe it's a bug in talon, as xprop shows the correct name, but I need to retest on the latest version

Comment (fidgetingbits) [reply apps/linux/keepassx.talon:4]: fWIW it still doesn't match on the latest version talon for some reason, but i'm not sure what/where to print from a ui.register() callback to see what talon is actually seeing the app string as.

Comment (knausj85) [reply apps/linux/keepassx.talon:4]: The easiest way is to have a command that dumps the focused app info to the console, or perhaps the clipboard. (clip.set_text())

debug app:
 print(app.name())
 print(app.bundle())
 print(app.executable())

Comment (fidgetingbits) [reply apps/linux/keepassx.talon:4]: thanks

```
2020-05-14 09:23:13    IO <pid 2181004>
2020-05-14 09:23:13    IO 
2020-05-14 09:23:13    IO 
```

Definitely a talon bug I think. So in the meantime we can just use the `win.title` if you were fine with that

Comment (fidgetingbits) [reply apps/linux/keepassx.talon:4]: Filed as https://github.com/talonvoice/beta/issues/62

Comment (fidgetingbits) [reply apps/linux/keepassx.talon:4]: bug is fixed in 1280, so will update the rules soon

**Pull request merged**